apps/cde-ui/src/app/components/visual-card/visual-card.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="visual-card" *ngFor="let card of cardData">
```

Comment:
Please switch to the new control flow syntax: https://angular.dev/guide/templates/control-flow